cms/static/js/spec/video/transcripts/videolist_spec.js

```diff
@@ -8,7 +8,9 @@ define(
     ],
 function ($, _, Utils, VideoList, MetadataModel, AbstractEditor, sinon) {
     'use strict';
-    describe('CMS.Views.Metadata.VideoList', function () {
+    // TODO: reenable these specs, using AjaxHelpers and the correct pattern
+    // to avoid flakiness. See TE-970 for details.
+    xdescribe('CMS.Views.Metadata.VideoList', function () {
```

Comment:
This is a lot of tests to disable in one shot. Do you plan to submit a ticket to have someone address this? I'm not sure that I'm comfortable with the trade off here. Can we leave this enabled and deal with the flakiness, or have someone do the refactoring to use AjaxHelpers?